src/awx_plugins/credentials/github_app.py

```diff
@@ -156,6 +157,28 @@ def _is_client_id(client_id_candidate: str) -> bool:
     return True
 
 
+def _is_iv1_prefix(client_id_candidate: str) -> bool:
+    """Check if it starts with Iv1 (not Iv10, Iv100, etc.)."""
+    known_v1_prefix = 'Iv1'
+
+    if not client_id_candidate.startswith(known_v1_prefix):
+        return False
+
+    client_id_candidate_trailer = client_id_candidate.removeprefix(
```

Comment:
This should unbreak the CI
```suggestion
```

(*I think* it's GitHub's new experimental review interface that might've corrupted the patch)